iib/web/static/api_v1.yaml

```diff
@@ -838,6 +838,14 @@ components:
             various modifications in the operator metadata embeded in the operator bundle
             image, e.g. change the container registry, or the operator package name.
           example: 'openshift-operators'
+        registry_auths:
+          type: object
+          description: >
+            Provide the dockerconfig.json for authentication to private registries
```

Comment:
Let's state here that non-auth information in the dockerconfig.json is not allowed.

Reply:
Changed